services/cinema_hall.py

```diff
@@ -0,0 +1,19 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
```

Comment:
You could indicate that you return the `QuerySet` that consists of `CinemaHall` instances in the return type annotation. The same is applicable to other places where you indicate `QuerySet` as a return type annotation, fix that.